Fix/10 manually add recipes

danielroehrig opened the pull request:
Reverted back to my original REST Endpoint that used PUT for updating a recipe. Since I introduced issue #10, I tried to fix it without losing the usage of PUT as an updating method. There's a new controller 'create' and route POST /recipes and some JavaScript glue code.

Apologies for my IDE that somehow "cleaned up" a lot of whitespace and empty lines. I only saw it after I pushed the changes to my fork.

mrzapp commented:
@danielroehrig I already fixed it in v0.5.1, could you check the latest code in master and see if it conflicts with the REST API in any way? If it doesn't, I suggest we keep it as is.

danielroehrig commented:
I checked out the 0.5.1 branch and saw that you used PUT to create a new 
recipe. PUT should only be used for updating resources, not creating new 
ones. That's why I added the POST route.

Am 9. Oktober 2019 07:21:56 schrieb Jeppe Zapp <notifications@github.com>:
> @danielroehrig I already fixed it in v0.5.1, could you check the latest 
> code in master and see if it conflicts with the REST API in any way? If it 
> doesn't, I suggest we keep it as is.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

mrzapp commented:
Alrighty. I've made you a collaborator now, so feel free to merge your changes into the develop branch
